Client/Frontend/Home/MessageCard/HomepageMessageCard.swift

```diff
@@ -234,8 +225,7 @@ extension HomepageMessageCardCell: ThemeApplicable {
         bannerTitle.textColor = theme.colors.textPrimary
         descriptionText.textColor = theme.colors.textPrimary
         dismissButton.imageView?.tintColor = theme.colors.textPrimary
-        ctaButton.backgroundColor = theme.colors.actionPrimary
-        ctaButton.setTitleColor(theme.colors.textInverted, for: .normal)
+        ctaButton.applyTheme(theme: theme)
```

Comment:
We shouldn't need to have to call `ctaButton.applyTheme(theme: theme)` manually, it will normally be called for us through the `Themeable` protocol, where all subviews that are `ThemeApplicable` will get their theme re-applied. So we can remove this line, but great work!

Reply:
Ok, so I realized we shouldn't remove entirely `ctaButton.applyTheme(theme: theme)`, but rather move it. It can be moved below `applyTheme(theme: theme)`. This way it will be called once on the configuration of the views, but then if theme changes dynamically while that view is already showing then as I said the apply theme for that button will be called from the protocol. 

Thank you!

Reply:
Hi @lmarceau, Thank you for reviewing. Yeah, It seems like we shouldn't remove the `ctaButton.applyTheme(theme: theme)` function entirely. As suggested I have moved it below `applyTheme(theme: theme)` in the configure function.

Thank You!